cmd/notary/keys.go

```diff
@@ -97,7 +99,12 @@ func (k *keyCommander) GetCommand() *cobra.Command {
 			"Required for timestamp role, optional for snapshot role")
 	cmd.AddCommand(cmdRotateKey)
 
-	cmd.AddCommand(cmdKeyImportTemplate.ToCommand(k.importKeys))
+	cmdKeysImport := cmdKeyImportTemplate.ToCommand(k.importKeys)
+	cmdKeysImport.Flags().StringVarP(
+		&k.keysImportRole, "role", "r", "", "Role to import key with - Notary can use this to infer the path to store the key. A specified path will take precedence")
+	cmdKeysImport.Flags().StringVarP(
```

Comment:
nit: can we rephrase both of these flag descriptions to something shorter like:

`Role to import key with, if a role is not already given in a PEM header`

I think the pathing logic can be explained from informative error messages rather than adding complexity to the CLI flags, since the error should depend on how the user imports the keys (straight from openssl vs. from `notary key export`)